### DISCUSSION TITLE
Kratos SPA using ReScript React

### DISCUSSION THREAD

**Discussion by allancalix** [ory/kratos#1623] [Show and tell]
I implemented the browser auth flows in ReScript React, there are plain JavaScript files commited alongside the ReScript in the [source](https://github.com/allancalix/kratos-ui). If anyone else pursues a UI using ReScript the [bindings](https://github.com/allancalix/kratos-ui/blob/main/src/Bindings/Kratos.res) can be useful to getting started with the JavaScript Kratos client library.

Here is the login page:
![login](https://github.com/allancalix/kratos-ui/blob/main/docs/img/login.png?raw=true)

**Comment by aeneasr** [comment]
Wow, that's awesome! @vinckr can we showcase this in the docs?
